### PR TITLE
Update default resource interpreter documents for PodDisruptionBudget

### LINKED ISSUE #287: Update interpreter documents as per #2997
**What would you like to be added**:
Update documents as per https://github.com/karmada-io/karmada/pull/2997 which introduced a default interpreter(`AggregateStatus`) for `PodDisruptionBudget(policy/v1)`.

An item should be added to [this doc](https://karmada.io/docs/next/userguide/globalview/customizing-resource-interpreter#aggregatestatus).

**Why is this needed**:

### PR DESCRIPTION
**Pull request by RainbowMango**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:
Update documents as per https://github.com/karmada-io/karmada/pull/2997 which introduced a default interpreter(`AggregateStatus`) for `PodDisruptionBudget(policy/v1)`.

**Which issue(s) this PR fixes**:
Fixes #287 

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by RainbowMango** [comment]
/assign @XiShanYongYe-Chang @Poor12 

@Poor12 You can see that the build is successful now, as a contrast, #287 still can't build now. Please check if there is something wrong with the patch and push it again.

**Comment by XiShanYongYe-Chang** [comment]
中文的版本是否需要同步修改？

**Comment by RainbowMango** [comment]
Good point, done.

File: docs/userguide/globalview/customizing-resource-interpreter.md [hunk 1]
```diff
@@ -79,6 +79,7 @@ Supported resources:
 - StatefulSet(apps/v1)
 - Pod(v1)
 - PersistentVolumeClaim(v1)
+- PodDisruptionBudget(policy/v1)
```
Comment (XiShanYongYe-Chang) [inline docs/userguide/globalview/customizing-resource-interpreter.md:82]: We also introduced `InterpretStatus` and `InterpretHealth` for PDB.

Comment (RainbowMango) [reply docs/userguide/globalview/customizing-resource-interpreter.md:86]: done.

**Pull request merged**